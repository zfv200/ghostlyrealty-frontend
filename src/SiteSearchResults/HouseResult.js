import React from 'react'

const HouseResult = (props) => {
  return (
    <div>
      <h1>{props.name}</h1>
      <img alt={`front shot of ${props.name}`} src={props.image_url}/>
    </div>
  )
}

export default HouseResult
