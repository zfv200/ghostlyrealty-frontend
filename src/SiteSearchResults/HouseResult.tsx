import React from 'react'
import styles from './HouseResult.css.js'


interface HouseResultStatelessProps {
  name: string,
  image_url: string
}

const HouseResult: React.SFC<HouseResultStatelessProps> = (props) => {
  return (
    <div>
      <h1>{props.name}</h1>
      <img style={styles.image} alt={`front shot of ${props.name}`} src={props.image_url}/>
    </div>
  )
}

export default HouseResult
