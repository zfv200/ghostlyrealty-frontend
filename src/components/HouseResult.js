import React from 'react'

const HouseResult = (props) => {
  return (
    <div>
      {props.name}
    </div>
  )
}

export default HouseResult
