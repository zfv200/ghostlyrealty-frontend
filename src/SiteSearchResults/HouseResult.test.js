import React from 'react'
import ReactDOM from 'react-dom'
import HouseResult from './HouseResult'

import { shallow } from 'enzyme'

const result = {
  address: null,
  burial_ground: false,
  featured: true,
  id: 1,
  image_url: "https://media.cntraveler.com/photos/580e6787e62940c66bd2be0d/master/w_820,c_limit/winchester-mystery-house-san-jose-GettyImages-110714519.jpg",
  name: "Winchester Mystery House",
  rooms: null,
  solo_haunt: false
}

it('renders without crashing', ()=>{
  const component = shallow(<HouseResult {...result}/>)

  expect(component).toMatchSnapshot()
})
