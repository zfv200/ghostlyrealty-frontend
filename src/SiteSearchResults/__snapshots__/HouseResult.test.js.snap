// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h1>
    Winchester Mystery House
  </h1>
  <img
    alt="front shot of Winchester Mystery House"
    src="https://media.cntraveler.com/photos/580e6787e62940c66bd2be0d/master/w_820,c_limit/winchester-mystery-house-san-jose-GettyImages-110714519.jpg"
  />
</div>
`;
